config/v1/types_apiserver.go

```diff
@@ -30,6 +30,15 @@ type APIServerSpec struct {
 	// - ConfigMap.Data["ca-bundle.crt"] - CA bundle.
 	// +optional
 	ClientCA ConfigMapNameReference `json:"clientCA"`
+	// extraCORSAllowedOrigins is a field of allowed CORS that should be added on top of the basic CORS needed by the
```

Comment:
The "added on top of" bit sounds like implementation detail.

Reply:
> The "added on top of" bit sounds like implementation detail.

The doc here needs to describe that this list is in addition to other values.  You can suggest different wording, but the intent must be there.

Reply:
I like the "not fully authoritative" wording David had in a later comment.